examples/run_CPMG.py

```diff
@@ -2,24 +2,28 @@
 CPMG
 ====
 
+..image:: CPMG_ppg.jpg
+
 This script will perform a standard CPMG experiment. 
 In order to form a symmetric echo, a padding time is added before 
-and after your tau through a series of delays. 
+and after your tau. Both the initial ninety pulse and 
+subsequent 180 pulses are phase cycled together,
+with a two step phase cycle on the ninety pulse (see
+:func: apply_cycles in SpinCore_pp.py)
```

Comment:
you're missing backticks here surrounding the apply_cycles

Reply:
I think fixed? check syntax